src/winter_onboarding_2021/fleet_management_service/routes.clj

```diff
@@ -0,0 +1,9 @@
+(ns winter-onboarding-2021.fleet-management-service.routes
+  (:require [bidi.ring :as br]
+            [ring.middleware.json :refer [wrap-json-response]]
+            [winter-onboarding-2021.fleet-management-service.handler :as handler]))
+
+(def routes [["/cabs"] handler/fetch-all-cabs])
+
+(def ring-handler (-> (br/make-handler routes)
```

Comment:
Also add the request logging middleware, so we can see incoming requests when working in dev.

Reply:
Done

Reply:
Why are we adding middlewares in multiple places?